docker/Dockerfile

```diff
@@ -191,18 +191,16 @@ FROM dependencies-dev-python as build-docs
 # Copy docs dependencies to build the rendered docs
 COPY mkdocs.yml /source/
 COPY docs /source/docs
+COPY scripts /source/scripts
 
 # Build the rendered docs, this ensures that the docs are in the final image.
-RUN mkdocs build --no-directory-urls
```

Comment:
Is the entire `build-docs` layer irrelevant now?

Reply:
I don't know... I am not really sure what the layering buys us. I see the docs on top of the docs, but tbh don't understand it all.